README.md

```diff
@@ -392,22 +392,41 @@ your-secretRootKey-value:
+| `rotationFrequency` | string | Indicates the interval between credential rotations. The actual frequency may be longer than the one you specified.  See details under **Notes** below. | "m", "h"                                                                                                                                                |                                                                                                                                                                |
+| `rotatedBindingTTL`   |  string | Specifies for how long to keep the old `ServiceBinding`. The actual TTL may be longer than the one you specified.  See details under **Notes** below.   | "m", "h"                                                                                                                                                |   
 
-**Note:**<br> It isn't possible to enable automatic credentials rotation to an already-rotated `ServiceBinding` (with the `services.cloud.sap.com/stale` label).
+### Example
+In the following example, the `ServiceBinding` resource is configured to rotate credentials every 25 days and to keep the old `ServiceBinding` for 2 days before deleting it.
+```yaml
+apiVersion: services.cloud.sap.com/v1
+kind: ServiceBinding
+metadata:
+  name: sample-binding
+spec:
+  serviceInstanceName: sample-instance
+  credentialsRotationPolicy:
+    enabled: true
+    rotatedBindingTTL: 48h
+    rotationFrequency: 600h
+ ```
+
+**Notes**
+* **The `credentialsRotationPolicy` is evaluated and executed during the [control loop](https://kubernetes.io/docs/concepts/architecture/controller/) which runs on every update or during
+  a full reconciliation process. This means that the actual rotation time may potentially be greater than the specified duration in 'rotationFrequency'.**
+* The original `ServiceBinding` and `Secret` will always contain the newest BindingID and credentials, old BindingID and credentials are backed up in a new `ServiceBinding` and `Secret` and will be deleted according to the configured TTL. The name of the backup `ServiceBinding` is '<original-binding-name>-guid'
+* To see when the service binding was last rotated, refer to the `status.lastCredentialsRotationTime`.
+* The credentials expiration time is determind by the service broker and is not affected by the `credentialsRotationPolicy` settings.
+* To initiate an instant rotation (regardless to the configured `rotationFrequency` value), simply add the `services.cloud.sap.com/forceRotate: "true"` annotation to the `ServiceBinding`. Note that the prerequisite for the force action is that credentials rotation enabled field is set to true.
+* It isn't possible to enable automatic credentials rotation to an already-rotated `ServiceBinding` (with the `services.cloud.sap.com/stale` label).
```

Comment:
what does it means?
that the "old" binding can be updated with `enabled: true`? is it removed by the operator when the binding is renamed?

If this is the case, it can be part of the explanation.. when it is mentioned the original binding is renamed, it can noted that it cannot be updated and cred policy cannot be disabled.